internal/cephfs/core/volume.go

```diff
@@ -231,7 +231,6 @@ func (s *subVolumeClient) CreateVolume(ctx context.Context) error {
 
 	opts := fsAdmin.SubVolumeOptions{
 		Size: fsAdmin.ByteCount(s.Size),
-		Mode: modeAllRWX,
```

Comment:
remove `modeAllRWX` variable which is of no use as we are not setting the mode anymore.

Reply:
Sure @Madhu-1 ..